servicebroker/test/routes/binding_test.js

```diff
@@ -0,0 +1,204 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var supertest = require("supertest");
+var expect = require('chai').expect;
+var nock = require('nock');
```

Comment:
Liu Ying and I do some investigation about sinon and did not find the way it work with supertest. Then we found "nock" is easy to use and fulfill our request, so we use "nock".